package.json

```diff
@@ -128,6 +128,7 @@
     "redux-saga": "^0.11.0",
     "regenerator-runtime": "^0.9.5",
     "striptags": "^2.1.1",
+    "styled-components": "^1.1.2",
```

Comment:
lets hold off on this until @JakeDawkins PR is merged / we have more time for a throughout investigation. I led you wrong in the JIRA task when I mentioned it, I was just meaning this long term could be a good use case for it with react-native support